src/main/java/racing/InputView.java

```diff
@@ -0,0 +1,31 @@
+package racing;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final int INPUT_THRESHOLD = 2;
+    public static int car() {
+        System.out.println("자동차 대수는 몇 대 인가요? (2 이상의 자연수 입력)");
+        Scanner scanner = new Scanner(System.in);
+        int car_count = scanner.nextInt();
+        isCorrectInput(car_count);
+
+        return car_count;
+    }
```

Comment:
```suggestion
        Scanner scanner = new Scanner(System.in);
        int carCount = scanner.nextInt();
        isCorrectInput(carCount);

        return carCount;
    }
```
스네이크 케이스를 사용하는 것도 좋습니다.

다만 일관성이 지켜져야 가능할 것 같아요
나머지 다른 부분들도 카멜케이스를 사용하셨으니 car_count도 카멜케이스에 맞춰주는 것이 좋겠습니다.